learning/tour-of-beam/frontend/.metadata

```diff
@@ -0,0 +1,42 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We should not have the license in generated files as they will get overwritten.

`build.gradle.kts` contains a list of files to skip during the license check. It ignores this file and `pubspec.lock` for Playground, and I added these two files for `tour_of_beam`. Change that to `tour-of-beam`, and remove the license from this file.

Reply:
is `build.gradle.kts` checked into source control? Seems like it's a file unique to IntelliJ.